doc/api_assets/style.css

```diff
@@ -1,3 +1,50 @@
+/*--------------------- CSS Variables ----------------------------*/
+:root {
+  --black: #000000;
+  --black1: #090c15;
+  --black2: #2c3437;
+  --white: #ffffff;
+  --white-smoke: #f2f2f2;
+  --red1: #d60027;
+  --red2: #d50027;
+  --red3: #ca5010;
+  --green1: #43853d;
+  --green2: #5a8147;
+  --green3: #454545;
+  --green4: #99cc7d;
+  --gray1: #707070;
+  --gray2: #b4b4b4;
+  --gray3: #cccccc;
+  --gray4: #040404;
+  --gray5: #7a7a7a;
+  --gray6: #333333;
+  --gray7: #c1c1c1;
+
+  --color-brand-primary: var(--gray6);
+  --color-brand-secondary: var(--green1);
+  --color-text-primary: var(--gray6);
+  --color-text-secondary: var(--green2);
+  --color-fill-app: var(--white);
+  --color-text-nav: var(--gray3);
+  --highlight-background-color: var(--white-smoke);
+}
+
+.dark-mode {
+    --color-fill-app: var(--black1);
+    --color-text-primary: var(--white);
```

Comment:
```suggestion
    --color-text-primary: var(--white);
    --color-links: var(--green4);
```

Reply:
```suggestion
    --color-text-primary: var(--white);
    --color-links: var(--green5);
```

Reply:
```
--color-fill-app: #090c15;
--color-fill-body: #090c15;
--color-fill-canvas: #090c15;
```

Aduh, check to make sure that they align w/ the front end once you get the chance.

Reply:
yup, this is inspired from nodejs.dev